src/reflect/value.go

```diff
@@ -1595,6 +1595,11 @@ func (v Value) Len() int {
 	case String:
 		// String is bigger than a word; assume flagIndir.
 		return (*unsafeheader.String)(v.ptr).Len
+	case Ptr:
+		if v.Elem().Kind() == Array {
```

Comment:
Maybe `return v.Elem().Len()`?

Reply:
this will need more check in kind == ARRAY